Altis_Life.Altis/config/Config_Loadouts.hpp

```diff
@@ -0,0 +1,84 @@
+/*
+    class PLAYERSIDE { // PLAYERSIDE can be: WEST (for cops), CIV (for civ/reb), GUER (for medics), EAST (for opfor)
+        // NOTES:
+        //     empty array means that nothing will be add on players
+        //     if you put more than a uniform on the CIV's class, they will be selected randonly,
+        //         otherwise, for the other teams, player will get the uniform related to his level
+
+        itemType[] = { // itemType can be: uniform, vest, backpack, weapon, items or linkedItems
```

Comment:
It doesn't look like you're using the conditions for `linkedItems` although it states in this comment that it is supported

Reply:
what do you guys think about it ? Should linkedItems get the ability to run under an specific condition or not ?

Reply:
There is no reason to not do it (no extra effort on your part), can't answer that question for the end user